packages/idyll-cli/src/index.js

```diff
@@ -126,7 +126,7 @@ const idyll = (options = {}, cb) => {
       // Leaving the following timing statement in for backwards-compatibility.
       if (opts.debug) console.time('Build Time');
 
-      pipeline.build(opts, paths, resolvers)
+      return pipeline.build(opts, paths, resolvers)
```

Comment:
I'm not sure we can do this without making an API change, since it short circuits the later `return this`. I'll keep searching for the root cause of the issue here.

Reply:
Ah you're right, I thought the `this` was part of the top-level `idyll` function return. Bleh